tools/c7n_azure/tests/cassettes/FunctionAppUtilsTest.test_publish_functions_package_consumption.yaml

```diff
@@ -126,6 +126,29 @@ interactions:
       x-ms-request-server-encrypted: ['true']
       x-ms-version: ['2018-03-28']
     status: {code: 201, message: Created}
+- request:
```

Comment:
Why did you re-record existing cassette? With this change it shouldn't be required..

Reply:
Using ```create_blob_from_stream``` appends ```?comp=blocklist``` to the end of the URL request. Added the additional request to compensate for this. 

Per your suggestion I will pass in the count to the function call to see if this removes the additional query variable. From there I can remove this additional request from the cassette